lib/moonshot/controller.rb

```diff
@@ -82,8 +85,13 @@ def doctor
 
     def ssh
       run_plugins(:pre_ssh)
-      stack.ssh
-      run_plugins(:post_ssh)
```

Comment:
This was removed because it was never reached, `Stack#ssh` does an exec call, which replaces Ruby with SSH.